### PR TITLE
Imp move get stats investment generation and remove select

### PR DESCRIPTION
**Pull request by MariteSomEnergia**
**Goal**

Move function get_stats_investment_generationkwh from som_inversions/account_account.py to  som_generationkwh/investment.py and remove SELECT statement.

Function created with some unit test related.

### CODE REVIEW

File: som_generationkwh/investment.py [hunk 1]
```diff
@@ -1983,6 +1983,29 @@ def error(message):
 
         return invoice_id, errors
 
+    def get_stats_investment_generation(self, cursor, uid, context=None):
+        """
+        Returns a list of dict with investment GenerationKwh statistics:
+            res: {'account': account code
```
Comment (oriolpiera) [inline som_generationkwh/investment.py:1989]: Perfecte, super elegant. Només adaptaria el comentari aquest a la nova resposta, sense la key 'account'.

Un cop fet el canvia, pots fer:
git commit --amend
git push -f

Per no haver de fer un altre commit.

File: som_generationkwh/investment.py [hunk 2]
```diff
@@ -1983,6 +1983,29 @@ def error(message):
...
+                  'amount': Total investment in Generation without amortizations}
+            params: No params
+        """
+
+        result = []
+
+        if not context:
+            context = {}
+
+        socis_ids = self.search(cursor, uid, [('emission_id.type', '=', 'genkwh'), ('last_effective_date', '=', False)])
```
Comment (oriolpiera) [inline som_generationkwh/investment.py:2000]: O.GenerationkwhInvestment.search([('emission_id.type', '=', 'genkwh'), ('last_effective_date', '>', datetime.datetime.now().strftime('%Y-%m-%d'))])

File: som_generationkwh/investment.py (hunk 1 above)
Comment (FranciscoCubero) [reply som_generationkwh/investment.py]: fet al següent commit

**Comment by FranciscoCubero** [comment]
@oriolpiera no es pot testejar adequadament des de webapps, mergejem a master i apliquem a productiu dilluns. A testing ha funcionat, valors similars.

**Pull request merged**